pkgs/development/python-modules/rpds-py/default.nix

```diff
@@ -35,6 +36,7 @@ buildPythonPackage rec {
     rustPlatform.maturinBuildHook
     cargo
     rustc
+    python3
```

Comment:
I don't think this is right, there should already be a specific python interpreter in `nativeBuildInputs`

Reply:
Correct. This is not needed.